db/silkworm/db/util.cpp

```diff
@@ -79,6 +79,20 @@ Bytes log_key(uint64_t block_number, uint32_t transaction_id) {
     return key;
 }
 
+std::optional<ByteView> find_value_suffix(mdbx::cursor& table, ByteView key, ByteView value_prefix) {
+    auto data{table.lower_bound_multivalue(to_slice(key), to_slice(value_prefix), /*throw_notfound*/ false)};
+    if (!data) {
+        return std::nullopt;
+    }
+    ByteView value{from_slice(data.value)};
```

Comment:
Why create a new byteview when data.value implements .startswith() ?
Just asking

Reply:
Oh I see ... nevermind ... misread the code